src/diffusers/loaders.py

```diff
@@ -1224,6 +1226,11 @@ def load_lora_weights(
             adapter_name=adapter_name,
             _pipeline=self,
         )
+        if not USE_PEFT_BACKEND:
```

Comment:
would be great to do this maybe in a follow up PR

Reply:
Yeah in v2, preferably.